README.md

```diff
@@ -9,9 +9,13 @@ The racing track is described by a series of checkpoints that you have to roll t
 
 
 ## Description of the grpc interface
-The idea behind the interface is that you can retrieve a view of the world around the sphere under your control and apply a force in the x/z plane to change the movement of the sphere.
+The idea behind the interface is that you can retrieve a view of the world around the sphere under your control and apply a force in the x/z direction to change the movement of the sphere.
 The view of the world (`getState`) contains the following fields:
-* `surrounding`: a list of (`height`, `kind` (ground type)) tuples describing the 9x9m surrounding of the player with 1 sample per meter. The first entry describes the world at (player.x - 4, player.z -4), the second at (player.x - 3, player.z - 4). If `kind == -1`, the coordinate is not part of the world.
+* `surrounding`: a list of (`height`, `kind` (ground type)) tuples describing the 9x9m surrounding of the player with 1 sample per meter. If `kind == -1`, the coordinate is not part of the world. The list structure is as follows:
```

Comment:
I'm curious about the data type which you've used for the surrounding. wouldn't be a matrix more intuitive?